sntools/genevts.py

```diff
@@ -89,11 +89,12 @@ def parse_command_line_options():
                         help="Transformation between neutrino flux inside SN and flux in the detector on Earth. \
                               Choices: %(choices)s. Default: %(default)s.")
 
-    choices = ("ibd", "es", "o16e", "o16eb", "c12e", "c12eb", "c12nc")
+    choices = ("ibd", "es", "o16e", "o16eb", "c12e", "c12eb", "c12nc", "ep")
     parser.add_argument("-c", "--channel", metavar="INTCHANNEL", choices=choices, default="all",
                         help="Interaction channels to consider. Currently supported: inverse beta decay (ibd), \
-                              electron scattering (es), nu_e + oxygen CC (o16e), nu_e-bar + oxygen CC (o16eb), \
-                              nu_e + carbon CC (c12e), nu_e-bar + carbon CC (c12eb) and nu + carbon NC (c12nc). \
+                              electron scattering (es), proton scattering (ep), nu_e + oxygen CC (o16e), \
+                              nu_e-bar + oxygen CC (o16eb), nu_e + carbon CC (c12e), nu_e-bar + carbon CC (c12eb), \
+                              and nu + carbon NC (c12nc). \
```

Comment:
Minor nitpick: The order of channels `choices` should be the same as the order in the help text.